espnet2/asr/frontend/adapters_utils/add_adapters.py

```diff
@@ -0,0 +1,80 @@
+import logging
+
+import numpy as np
+
+from espnet2.asr.frontend.adapters_utils.adapters.adapter_transformer import (
+    AdapterTransformerSentenceEncoderLayer,
+)
+from espnet2.torch_utils.model_summary import model_summary
+
+
+def add_adapters_wav2vec2(wav2vec2_model, adapter_down_dim, adapt_layers=[]):
```

Comment:
Sorry, where was this function used?

Reply:
sounds like a good place to put the documentation, do you want the current README to remain in the adapters folder after I intergrate it to the official docs?

Reply:
I am leaning to simply integrating the readme in the official docs. As far as I know, there are no examples in espnet to keep a readme for a single module.

Reply:
docs integrated!

Reply:
the current implementation follows 1's design I think